mm2src/coins/lightning/ln_events.rs

```diff
@@ -417,7 +424,7 @@ impl LightningEventHandler {
         );
         let db = self.db.clone();
         let platform = self.platform.clone();
-        let abort_handler = spawn_abortable(async move {
+        self.platform.spawner().spawn(async move {
```

Comment:
The updating channel to closed in db part of the `save_channel_closing_details` fn needs to be spawned with timeout https://github.com/KomodoPlatform/atomicDEX-API/blob/873d9a68c7d5e96716c5c859fca220611ddd32d7/mm2src/coins/lightning/ln_events.rs#L219 We can move the above part outside the function to do that. The rest will be executed again on coin re-activation here https://github.com/KomodoPlatform/atomicDEX-API/blob/873d9a68c7d5e96716c5c859fca220611ddd32d7/mm2src/coins/lightning/ln_events.rs#L152-L153 but as you can see it requires to get the closed channels from db first.

I guess I missed this when we discussed this part before :)

Reply:
I guess we can just spawn `save_channel_closing_details` function with timeout since `update_channel_to_closed` is the first line in it. I missed this much easier solution while writing the first comment :sweat_smile:

Reply:
Sure, will do

Reply:
Done